flexget/plugins/sites/torrentleech.py

```diff
@@ -121,48 +120,35 @@ def search(self, task, entry, config=None):
             categories = [categories]
         # If there are any text categories, turn them into their id number
         categories = [c if isinstance(c, int) else CATEGORIES[c] for c in categories]
-        filter_url = '/categories/%s' % ','.join(str(c) for c in categories)
+        filter_url = '/categories/{}'.format(','.join(str(c) for c in categories))
         entries = set()
         for search_string in entry.get('search_strings', [entry['title']]):
             query = normalize_unicode(search_string).replace(":", "")
-            # urllib.quote will crash if the unicode string has non ascii characters, so encode in utf-8 beforehand
-            url = ('https://v4.torrentleech.org/torrents/browse/index/query/' +
+            # urllib.quote will crash if the unicode string has non ascii characters,
+            # so encode in utf-8 beforehand
+
+            url = ('https://www.torrentleech.org/torrents/browse/list/query/' +
```

Comment:
i don't think you meant to make the url a tuple

Reply:
It's not a tuple.

Reply:
This will not make it into a tuple it is just a method to construct a multiline string.